### DISCUSSION TITLE
Renamed select command to toggle

### DISCUSSION THREAD

**Discussion by ralphee** [sxyazi/yazi#2078] [1. Q&A] [answered]
### What system are you running Yazi on?

Linux Wayland

### What terminal are you running Yazi in?

foot

### `yazi --debug` output

```Shell
https://bpa.st/UECQ
```

### Describe the question

Ever since the update to 4.0 I cannot use the space bar to select multiple files in one action.  Previously hitting the space bar multiple times would select the file / folder and move on to the next file /folder to be selected. 4.0 I now have to manually move / hover over the file / folder I would like to select. I Hope you can understand what I am getting at.

### Checklist

- [X] I have searched the existing discussions/issues

**Comment by AnirudhG07** [comment]
In your keymap.toml, can you share whats there for Space bar??
Are you using the default settings??

**Reply by ralphee** [reply]
> In your keymap.toml, can you share whats there for Space bar?? Are you using the default settings??

Thanks for getting back to me! I probably made some mistake when manually editing my config files after the upgrade. As per your request i have provided the config files:

[keymap.toml](https://bpa.st/TOZQ)

[yazi.toml](https://bpa.st/LIWA)

**Reply by AnirudhG07** [reply, marked as answer]
Yes your keymaps are wrong from space bar.
It should be this.
```
	{ on = "<Space>", run = [ "toggle", "arrow 1" ], desc = "Toggle the current selection state" },
```

Also I will recommend, the new feature of yazi is that you need not write the WHOLE keymap.toml, you can only have the keybindings you want to MODIFY as `prepend_keymaps`. Check out my [configs](https://github.com/AnirudhG07/dotfiles/tree/stow/yazi/.config/yazi).

I think this should ans your problem!
